src/routes/outcomes/tree.ts

```diff
@@ -16,6 +16,9 @@ type outcome = {
 	text: string
 }
 
+const doomer_hope_str = " Luckily, you've just learn this, as if it were a message from the future! And you are still on time to make a Pause happen. Good luck John Connor, the future is not set."
```

Comment:
I think the messages should link to an action page. Instead of adding it to the tree I'd add it to the view I think